launcher/launcher/main.go

```diff
@@ -72,6 +79,16 @@ func main() {
 		return
 	}
 
+	args := fmt.Sprintf("-output=%s", experimentDataFile)
+	exec.Command(binaryPath, args).Run()
+
+	e, err := experiments.New(path.Join(launcherfile.HostTmpPath, experimentDataFile))
+	if err != nil {
+		logger.Println("failed to read experiment file %w", err)
```

Comment:
If you are using the a format verb, you need to use logger.Printf. The format verb %w is typically used when returning an error however, so you don't need to wrap the err. Just use %v or:
`logger.Println("Failed to read experiment file: ", err)`

Reply:
Gotcha, thanks for the tip. I think the println is an artifact from previous statement, thanks for catching that,